core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -51,6 +52,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-disown"
     )
 
+    conf.sslConf match {
+      case Some(ssl) => {
+        cmdToLaunch ++ Array("-internal_security_config", ssl)
```

Comment:
Yes, basically if the H2O cluster is already running in secure mode then it's using some key stores. Those key stores have to be used also in SW instances (the External Reader and Writer) so the only option is for the user to manually provide them. We could make an endpoint in H2O fetching them but then again fetching key stores over unsecured channels misses the point.

So it's either you go auto or copy the files yourself. This is also how encryption in H2O core works if we're not running on top of hadoop.